cpp/dolfinx/fem/FiniteElement.cpp

```diff
@@ -115,30 +103,45 @@ FiniteElement<T>::FiniteElement(
       _entity_dofs(element.entity_dofs()),
       _entity_closure_dofs(element.entity_closure_dofs())
 {
-  // TODO: symmetric rank-2 symmetric tensors are presently constructed
-  // as rank-1 tensors, e.g. a rank-2 symmetric tensor in 3D is
-  // constructed as rank-1 with shape (6,). It should be really be
-  // shape=(3, 3) with block size 6.
-
-  // If element is blocked, check that base element is scalar
   if (value_shape and !element.value_shape().empty())
   {
     throw std::runtime_error("Blocked finite elements can be constructed only "
                              "from scalar base elements.");
   }
 
-  if (symmetric) // Consistency check for symmetric elements
-  {
-    if (!value_shape)
+  if (symmetric && value_shape){
```

Comment:
Run `clang-format`/

Reply:
Fixed. I'll still suggest to wait so I test this in more elaborate tests in dolfiny. TBH, I just fixed the apparent places and I do not see completely through if all works downstream (the code is a bit complicated).